### ISSUE TITLE
'view_tool_product' not found when I added a tool or clicked on view tools on product dashboard settings.

### ISSUE THREAD

**Issue by pmagnemi20** [DefectDojo/django-DefectDojo#6111]
### Discussed in https://github.com/DefectDojo/django-DefectDojo/discussions/6026

<div type='discussions-op-text'>

<sup>Originally posted by **worapojc** March 16, 2022</sup>
Hi all,

I've configured **SonarQube** in **Tools Configuration**.

```
API connection successful with message: You have access to 8 projects.
Tool Configuration successfully updated.
```
I tried to add SonarQube integration for the product via _Settings -> Add Tools_ to import from SonarQube project.
However, it returns HTTP 500. Please see the stacktrace that I copied from uWSGI service.

How do I solve this issue?

> django.urls.exceptions.NoReverseMatch: Reverse for 'view_tool_product' not found. 'view_tool_product' is not a valid view function or pattern name.
> ERROR:django.request:Internal Server Error: /product/1/tool_product/all
> Traceback (most recent call last):
>   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
>     response = get_response(request)
>   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
>     response = wrapped_callback(request, *callback_args, **callback_kwargs)
>   File "/app/./dojo/authorization/authorization_decorators.py", line 30, in _wrapped
>     return func(request, *args, **kwargs)
>   File "/app/./dojo/tool_product/views.py", line 50, in all_tool_product
>     return render(request, 'dojo/view_tool_product_all.html', {
>   File "/usr/local/lib/python3.8/site-packages/django/shortcuts.py", line 19, in render
>     content = loader.render_to_string(template_name, context, request, using=using)
>   File "/usr/local/lib/python3.8/site-packages/django/template/loader.py", line 62, in render_to_string
>     return template.render(context, request)
>   File "/usr/local/lib/python3.8/site-packages/django/template/backends/django.py", line 61, in render
>     return self.template.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 170, in render
>     return self._render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 162, in _render
>     return self.nodelist.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
>     bit = node.render_annotated(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
>     return self.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 150, in render
>     return compiled_parent._render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 162, in _render
>     return self.nodelist.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
>     bit = node.render_annotated(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
>     return self.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 62, in render
>     result = block.nodelist.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
>     bit = node.render_annotated(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
>     return self.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 315, in render
>     return nodelist.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
>     bit = node.render_annotated(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
>     return self.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 214, in render
>     nodelist.append(node.render_annotated(context))
>   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
>     return self.render(context)
>   File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 449, in render
>     url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
>   File "/usr/local/lib/python3.8/site-packages/django/urls/base.py", line 86, in reverse
>     return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
>   File "/usr/local/lib/python3.8/site-packages/django/urls/resolvers.py", line 698, in _reverse_with_prefix
>     raise NoReverseMatch(msg)</div>

**Comment by valentijnscholten** [comment]
what version are you running?

**Comment by pmagnemi20** [comment]
latest, 2.8.0

**Comment by StefanFl** [comment]
merged for release 2.9.0
